submarine-cloud/Makefile

```diff
@@ -34,20 +33,19 @@ LDFLAGS		:= -s -w \
 
 .PHONY: build
 build:
-	$(GOBUILD) -o ./bin/$(BINARY_NAME) -v cmd/operator/main.go
+	GOOS=${GOOS} GOARCH=amd64 $(GOBUILD) -o ./bin/$(BINARY_NAME) -v cmd/operator/main.go
 
 test:
-	$(GOTEST) -v ./...
+	#$(GOTEST) -v ./...
```

Comment:
Delete this line instead of add a comment "#"?

Reply:
Now, Test cases for submarine-cloud are not yet supported.
I will support it in next work.